torch/csrc/utils/python_numbers.h

```diff
@@ -132,3 +132,24 @@ inline std::complex<double> THPUtils_unpackComplexDouble(PyObject *obj) {
 
   return std::complex<double>(value.real, value.imag);
 }
+
+inline bool THPUtils_unpackNumberAsBool(PyObject* obj) {
+  //when obj is float(contains NaN, inf, -inf)
+  if (PyFloat_Check(obj)) {
+    return (bool)PyFloat_AS_DOUBLE(obj);
+  }
+
+  //when obj is complex number
+  if (PyComplex_Check(obj)) {
+    double real_val = PyComplex_RealAsDouble(obj);
+    double imag_val = PyComplex_ImagAsDouble(obj);
+    return !(real_val == 0 && imag_val == 0);
+  }
+  //when obj is other
```

Comment:
Don't need this comment.

Reply:
I'll delete all the comments that you pointed out.